interop/soak_tests.go

```diff
@@ -175,25 +175,26 @@ func DoSoakTest(ctx context.Context, soakConfig SoakTestConfig) {
 		MinValue:       0,
 	})
 	for _, worker := range soakWorkerResults {
-		totalIterations += worker.IterationsDone
+		totalSuccesses += worker.IterationsDone
```

Comment:
Should we rename the `IterationsDone` field, if it's only counting successful iterations, to make this kind of bug more obvious?